elements/howto-menu/howto-menu.js

```diff
@@ -0,0 +1,216 @@
+     * Returns true if the menu is currently hidden, false otherwise.
+     */
+    _isHidden() {
+      return this.getAttribute('aria-hidden') === 'true';
+    }
+
+    /**
+     * Returns a menu item coming after the one passed as an argument, or null.
+     */
+    _nextMenuItem(node) {
+      let next = node.nextElementSibling;
+      while (next) {
+        if (this._isMenuItem(node)) {
+          return next;
+        }
+        next = next.nextElementSibling;
+      }
+      return null;
+    }
+
+    /**
+     * Returns a menu item coming before the one passed as an argument, or null.
+     */
+    _previousMenuItem(node) {
+      let prev = node.previousElementSibling;
+      while (prev) {
+        if (this._isMenuItem(node)) {
+          return prev;
+        }
+        prev = prev.previousElementSibling;
+      }
+      return null;
+    }
+
+    /**
+     * Opens the menu.
+     */
+    _open() {
+      this.firstMenuItem.setAttribute('tabindex', 0);
+      this.firstMenuItem.focus();
+    }
+
+    /**
+     * Closes the menu.
+     */
+    _close() {
+      const children = Array.from(this.children);
+      children.forEach(el => {
+        el.setAttribute('tabindex', -1);
+      });
+      this._opener.focus();
+    }
+
+    /**
+     * Makes the element unfocusable, as a reaction to blur event.
+     * @param {Event} event Blur event.
+     */
+    _handleBlur(event) {
+      if (this._isMenuItem(event.target)) {
+        event.target.setAttribute('tabindex', -1);
+      }
+    }
+
+    /**
+     * Hanldes keyboard interaction.
+     * @param {Event} event Keydown event.
+     */
+    _handleKeyDown(event) {
+      let el = event.target;
+      switch (event.keyCode) {
+        case KEYCODE.DOWN:
+          // If arrow down, move to next item. Wrap if necessary.
+          event.preventDefault();
+          this._moveFocus(el, this._nextMenuItem(el) || this.firstMenuItem);
+          break;
+
+       case KEYCODE.UP:
+         // If arrow up, move to previous item. Wrap if necessary.
+         event.preventDefault();
+         this._moveFocus(el, this._previousMenuItem(el) || this.lastMenuItem);
+         break;
+
+       case KEYCODE.ESCAPE:
+         // If escape, exit the menu.
+         event.preventDefault();
+         this.setAttribute('aria-hidden', 'true');
+         break;
+
+       default:
+         break;
+      };
+      // If letter key, move to an item which starts with that letter.
+      if (event.keyCode > 64 && event.keyCode < 91) {
+        for (let i = 0, child; child = this.children[i]; i++) {
+          if (child.innerText.trim()[0] === event.key) {
+            event.preventDefault();
+            this._moveFocus(el, child);
+          }
+        }
+      }
+    }
+
+    /**
+     * Sets up keyboard interactions for menu and its items.
+     */
+    connectedCallback() {
+      // Make children unfocusable by default.
+      const children = Array.from(this.children);
+      children.forEach(el => {
```

Comment:
I think having a `reset()` method similar to dash-tabs might be good here.
https://github.com/GoogleChrome/howto-components/blob/master/elements/howto-tabs/howto-tabs.js#L202

That way we could be consistent across elements.